rules/add-group-claim-to-token.js

```diff
@@ -0,0 +1,50 @@
+function (user, context, callback) {
+
+    var request   = require('request');
+    var namespace = "https://dev.xyz/claims/";  // For custom claims, you must define a namespace for oidc compliance. See https://auth0.com/docs/api-auth/tutorials/adoption/scope-custom-claims
```

Comment:
No, it's arbitrary.  If I've understood the [docs](https://auth0.com/docs/api-auth/tutorials/adoption/scope-custom-claims) correctly it could be `https://fishandchips/claims`.

Reply:
Since it is arbitrary, should it be something a bit more relevant, like `https://dev.mojanalytics.xyz/claims/`? Does the namespace need to be used to retrieve the claims from the token?